pallets/randomness/src/lib.rs

```diff
@@ -189,6 +189,11 @@ pub mod pallet {
 	pub(crate) type RelayTime<T: Config> =
 		StorageValue<_, RelayTimeInfo<T::BlockNumber, u64>, ValueQuery>;
 
+	/// Ensures the mandatory inherent was included in the block
+	#[pallet::storage]
+	#[pallet::getter(fn inherent_included)]
+	pub(crate) type InherentIncluded<T: Config> = StorageValue<_, ()>;
```

Comment:
We need this "bool" because if the relay block and relay epoch do not change, then no storage will be updated even when the inherent is included (so we cannot use these storage changes as a sign the inherent was included). 

The purpose is just to ensure this pallet's inherent is included in every block.